docs/source/handbook/operations/in_flight.md

```diff
@@ -37,7 +37,7 @@ Once the [pre-flight operations](./pre_flight.md) are complete and the tests are
 
  8. If AVAPS is connected to aircraft data (which it should be during flight operations), the meteorological data from the aircraft is taken in automatically.
 
- 9. The dropsonde is now ready. Prepare the sonde for launching. First, disconnect the data cable and put the sonde in the re-conditioning-box. Click on `Continue`.
+ 9. The dropsonde is now ready. Prepare the sonde for launching. First, disconnect the data cable and put the sonde in the re-conditioning-box. Click on `Continue`. An error message might appear if the data cable is removed before clicking `Continue`. It may be solved by leaving the cable in, clicking `Continue`, then removing the cable.
```

Comment:
Do I understand correctly that contrary to what was mentioned, one should first click Continue and then disconnect the cable? Maybe it is better to put it directly that way then and mention that doing it the other way may result in an error message. Additionally, any clue what the error message is, or what can be done to fix it?